core/src/main/scala/caliban/Rendering.scala

```diff
@@ -20,7 +20,9 @@ object Rendering {
               if (isBuiltinScalar(name)) None
               else
                 Some(
-                  s"""${renderDescription(t.description)}scalar $name${renderSpecifiedBy(t.specifiedBy)}""".stripMargin
+                  s"""${renderDescription(t.description)}\n${renderDirectives(
```

Comment:
According to the spec, the directives come after the name: https://spec.graphql.org/October2021/#sec-Scalars

Reply:
Ah, TIL. Thanks will fix.

Reply:
Also I think you don't need any space or line break, if you look at other usage of `renderDirectives`